### PR TITLE
Removed error

### PR DESCRIPTION
**Pull request by snehit31**
The PR error has been removed and they are fit for merging.

### CODE REVIEW

**Comment by snehit31** [comment]
Trying to remove the errors

File: _data/projects/data_science.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: data science
+desc: It is for data science enthusiasts like me
+site: https://github.com/snehit31/datasciencecoursera
+tags: 
+- datascience
+- rlanguage
+- python
+- machinelearning
+upforgrabs:
+name: datasciencecoursera
```
Comment (shiftkey) [inline _data/projects/data_science.yml:10]:
```suggestion
   name: datasciencecoursera
```

File: _data/projects/data_science.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+link: https://github.com/snehit31/datasciencecoursera
```
Comment (shiftkey) [inline _data/projects/data_science.yml:11]:
```suggestion
  link: https://github.com/snehit31/datasciencecoursera
```

**Review by shiftkey — CHANGES_REQUESTED** [review]
YAML is a fickle format at times - try this

**Pull request closed without merging**